yt_dlp/extractor/tiktok.py

```diff
@@ -242,7 +244,8 @@ def extract_addr(addr, add_meta={}):
                 'format_id': 'play_addr',
                 'format_note': 'Direct video',
                 'vcodec': 'h265' if traverse_obj(
-                    video_info, 'is_bytevc1', 'is_h265') else 'h264',  # TODO: Check for "direct iOS" videos, like https://www.tiktok.com/@cookierun_dev/video/7039716639834656002
+                    video_info, 'is_bytevc1', 'is_h265') else 'h264',
+                # TODO: Check for "direct iOS" videos, like https://www.tiktok.com/@cookierun_dev/video/7039716639834656002
```

Comment:
revert unnecessary/unrelated formatting change
```suggestion
                    video_info, 'is_bytevc1', 'is_h265') else 'h264', # TODO: Check for "direct iOS" videos, like https://www.tiktok.com/@cookierun_dev/video/7039716639834656002
```